src/mono/mono/component/marshal-ilgen.h

```diff
@@ -42,5 +58,8 @@ mono_emit_marshal_ilgen (EmitMarshalContext *m, int argnum, MonoType *t,
 
 void
 mono_marshal_ilgen_init_internal (void);
```

Comment:
`mono_marshal_ilgen_init_internal` appears to have 2 declarations in this file now.  I think it should have zero.